_sass/module/_hero.scss

```diff
@@ -44,7 +46,6 @@ $hero-breakpoint: $grid--medium;
     &::before {
       content: '';
       position: absolute;
-      background-color: $scrim-color;
```

Comment:
We probably just want to delete this whole pseudo element. It's only purpose was to provide the scrim.